README.md

```diff
@@ -2,13 +2,18 @@
 
 [![Stable Version](https://img.shields.io/pypi/v/basedpyright?logo=pypi)](https://pypi.org/project/basedpyright/) [![visual studio marketplace](https://img.shields.io/visual-studio-marketplace/d/detachhead.basedpyright?logo=visualstudiocode)](https://marketplace.visualstudio.com/items?itemName=detachhead.basedpyright) [![open VSX](https://img.shields.io/open-vsx/dt/detachhead/basedpyright?logo=vscodium)](https://open-vsx.org/extension/detachhead/basedpyright) [![nvim-lspconfig](https://img.shields.io/badge/nvim--lspconfig-grey?logo=neovim)](https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#basedpyright) [![sublime text](https://img.shields.io/packagecontrol/dt/LSP-basedpyright?logo=sublimetext)](https://packagecontrol.io/packages/LSP-basedpyright) [![Discord](https://img.shields.io/discord/948915247073349673?logo=discord)](https://discord.gg/7y9upqPrk2)
 
-Basedpyright is a fork of [pyright](https://github.com/microsoft/pyright) with various type checking improvements, improved vscode support and pylance features built into the language server.
+
+Basedpyright is a Python development tool that provides static analysis including type-checking and linting,
+along with IDE functionality of syntax highlighting and code-actions like import suggestions.
+
+This project was originally based on the [pyright](https://github.com/microsoft/pyright) project, but has implemented various enhancements to type checking, configuration, packaging, and the command-line interface.
+Additionally, providing improved IDE support and features built into the language server that were previously exclusive to the closed source pylance.

```

Comment:
- idk about calling it a linter. although there are some rules that are more linty than type checky, there's currently no distinction between them so its probably best to not make a distinction in the description
- what "packaging, and the command-line interface" means? "packaging" makes it sound like it has a compiler/bundler in it or something. and i havent touched the cli at all other than changing the installation process
what about this
```suggestion
Basedpyright is a Python type checker and language server, based on the [pyright](https://github.com/microsoft/pyright) project, but has implemented various enhancements to type checking, configuration and IDE integration using the language server protocol.

basedpyright provides improved IDE support and features built into the language server that were previously exclusive to microsoft's closed-source pylance vscode extension.
```